mesheryctl/pkg/utils/platform.go

```diff
@@ -36,78 +36,31 @@ var (
 	ReleaseTag      string
 )
 
-// ChangePlatform changes the platform specified in the current context to the specified platform
-func ChangePlatform(currCtx string, ctx config.Context) error {
-	ViperK8s.SetConfigFile(DefaultConfigPath)
-	err := ViperK8s.ReadInConfig()
+// UpdateConfigContext updates the given context in meshconfig
+func UpdateConfigContext(currCtx string, ctx *config.Context) error {
+	ViperMeshconfig.SetConfigFile(DefaultConfigPath)
+	err := ViperMeshconfig.ReadInConfig()
 	if err != nil {
```

Comment:
Didn't we consolidate writing a context struct to the config [somewhere here](https://github.com/meshery/meshery/blob/master/mesheryctl/internal/cli/root/config/config.go)?

There should be a function that handles this. Ideally we want a single context struct as the source of truth at any time and we should be writing that struct to the config through a single central function.

Reply:
Good point, it's better to shift this to config package as all such func should belong to that.

Reply:
Review now!